src/almanack/entropy.py

```diff
@@ -0,0 +1,39 @@
+"""
+This module finds the amount of entropy on a file based level
```

Comment:
Would there ever be a need for this module beyond the single function below? Consider generalizing this statement to help prepare for other work which may need to go here.

Reply:
I tried to think about this when creating the doc-string. I figured that down the line this module could have different calculation functions. I'm struggling to figure out how to get more general with this comment. What are your ideas?

Reply:
Thanks, mostly I wondered if there might be a way to generalize this as "software entropy" (at any level).